File: kotlin-ide/j2k/old/testData/fileOrElement/annotations/modifiersToAnnotationsFromPropertyAccessors.java
public class WithModifiersOnAccessors {
    private synchronized void methSync() {}
    protected strictfp void methStrict() {}

    private int sync = 0;
    public synchronized int getSync() { return sync; }
    public synchronized void setSync(int sync) { this.sync = sync; }

    public double strict = 0.0;
    public strictfp double getStrict() { return strict; }
}

File: nj2k/testData/newJ2k/annotations/modifiersToAnnotationsFromPropertyAccessors.kt
class WithModifiersOnAccessors {

    @get:Synchronized
    @set:Synchronized
    var sync = 0

    @get:Strictfp
    val strict = 0.0

    @Synchronized
    private fun methSync() {
    }

    @Strictfp
    protected fun methStrict() {
    }
}